.github/workflows/test.yml

```diff
@@ -10,10 +10,10 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        python-version: [ 3.7, 3.8, 3.9 ]  # latest release minus two
+        python-version: [ 3.9, 3.10, 3.11, 3.12 ]  # latest release minus two
```

Comment:
3.10 , 3.11, 3.12 should be wrapped with quotation mark, otherwise yaml will be incorrectly analyzed.